superset/datasets/schemas.py

```diff
@@ -42,26 +43,25 @@
 }
 
 
-def validate_python_date_format(value: str) -> None:
-    regex = re.compile(
-        r"""
-        ^(
-            epoch_s|epoch_ms|
-            (?P<date>%Y([-/]%m([-/]%d)?)?)([\sT](?P<time>%H(:%M(:%S(\.%f)?)?)?))?
-        )$
-        """,
-        re.VERBOSE,
-    )
-    match = regex.match(value or "")
-    if not match:
-        raise ValidationError([_("Invalid date/timestamp format")])
+def validate_python_date_format(value: str) -> bool:
+    if value in ("epoch_s", "epoch_ms", None):
```

Comment:
```suggestion
    if value in ("epoch_s", "epoch_ms"):
```

Per the type hint `value` cannot be `None`.